opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionLists.tsx

```diff
@@ -169,9 +172,16 @@ const ExclusionLists = () => {
     description: {
       id: 'description',
       label: t_i18n('Description'),
-      percentWidth: 30,
+      percentWidth: 20,
       isSortable: false,
     },
+    lineNumber: {
+      id: 'NumberOfElements',
+      label: t_i18n('Number of elements'),
+      percentWidth: 10,
```

Comment:
please harmonize the column width. maybe 15 here, or 10 too on createdat, enabled andentity types